app/src/components/v-menu/v-menu.vue

```diff
@@ -4,6 +4,7 @@
 			ref="activator"
 			class="v-menu-activator"
 			:class="{ attached }"
+			@focusout="deactivate"
```

Comment:
How does this bubble up when you're in a nested menu? There's a handful of places where we have menus in menus (like the datetime or color interfaces). Does a child-focusout cause the parent to lose focus too?

Reply:
Oh right.. Thanks for mentioning the "menus in menus", didn't thought of that. Will try to look into it & see does it break or not (unfortunately most likely would from what I can think of).